src/utils.ts

```diff
@@ -594,3 +594,24 @@ export async function estimateCustomBridgeDepositL2Gas(
         l2Value: l2Value,
     });
 }
+
+export function encodeData(data: any[]): string {
+    const types = data.map(item => {
+        if (typeof item === 'number') {
+            return 'uint256';
```

Comment:
what if I need `uint128` or `int64`?

Reply:
good question. I thought the same while setting up this function too (another reason why I left this PR as draft). I would think now then that it is not possible to deduce these lower levels types without the developer specifying them either through an ABI or some form of input. This changes the syntax for constructing both the `Paymaster` and `abstractWallet` classes, considering they both can potentially have methods that serialise data.